### PR TITLE
Random dog

### PR DESCRIPTION
**Pull request by AchimBartscht**

### CODE REVIEW

File: pages/index.js [hunk 1]
```diff
@@ -13,6 +26,79 @@ export default function HomePage() {
       <LinkButton $fullWidth href="/quiz-page">
         Let&apos;s begin
       </LinkButton>
+      <Headline>Dog of the moment</Headline>
+      <StyledLink href={`dogs/${randomDog.name}`}>
+        <div>
```
Comment (ThomasKirst) [inline pages/index.js:31]: The `div` tag might not be needed here – or is it related to the `figure` animation?

File: pages/index.js [hunk 2]
```diff
@@ -13,6 +26,79 @@ export default function HomePage() {
...
+      opacity: 1;
+    }
+    40% {
+      opacity: 1;
+    }
+    100% {
+      width: 120%;
+      height: 120%;
+      opacity: 0;
+    }
```
Comment (ThomasKirst) [inline pages/index.js:75]: Please check the format here.

**Review by ThomasKirst — CHANGES_REQUESTED** [review]
Hi Achim 👋

Looks all good, I just left you two minor comments.

For now, I would say the solution is fine, but for the future I would reconsider the following:
- you´re loading more than 200 dogs from your API 😅
- in your client code you grab a random dog from the list
- this could be done on the server side already where you return only one random dog to the browser in order to save some bandwidth and waiting time for the user

Also, nice design 👍

– Thomas

File: pages/index.js (hunk 1 above)
Comment (gregorsart) [reply pages/index.js:31]: I think we need the extra div tag, but we will try it out without the `div` tag and when it still works we will remove it.

Comment (alinarub) [reply pages/index.js:31]: It works without the `div`, you're right :)

**Comment by alinarub** [comment]
> Hi Achim 👋
> 
> Looks all good, I just left you two minor comments.
> 
> For now, I would say the solution is fine, but for the future I would reconsider the following:
> 
> * you´re loading more than 200 dogs from your API 😅
> * in your client code you grab a random dog from the list
> * this could be done on the server side already where you return only one random dog to the browser in order to save some bandwidth and waiting time for the user
> 
> Also, nice design 👍
> 
> – Thomas

You're right but we're now disconnected from the API, we pulled all the data to our lib folder. Now via backend route we get all the dogs on the server and send it to the frontend. Of course we could do the calculation in the backend and then send the random dog to the frontend. But we're in the last two days of our bootcamp now, so we want to focus on implementing dark mode first. :)

**Review by NikRoe — APPROVED** [review]
LGTM 🐕

**Pull request merged**